tox.ini

```
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{310,311,312,dev}-test{,-recdeps,-alldeps,-oldestdeps,-devdeps,-devinfra,-predeps,-numpy123,-numpy124,-numpy125,-mpl334}{,-cov}{,-clocale}{,-fitsio}{,-noscipy}
+    py{310,311,312,dev}-test{,-recdeps,-alldeps,-oldestdeps,-devdeps,-mpldev,-devinfra,-predeps,-numpy123,-numpy124,-numpy125,-mpl363}{,-cov}{,-clocale}{,-fitsio}{,-noscipy}
```

Only image tests use mpldev . In regular CI, mpldev is folded into devdeps.
```suggestion
    py{310,311,312,dev}-test{,-recdeps,-alldeps,-oldestdeps,-devdeps,-devinfra,-predeps,-numpy123,-numpy124,-numpy125,-mpl363}{,-cov}{,-clocale}{,-fitsio}{,-noscipy}
```